app/scheduled/circulars/circularAction.ts

```diff
@@ -0,0 +1,14 @@
+/*!
+ * Copyright © 2023 United States Government as represented by the
+ * Administrator of the National Aeronautics and Space Administration.
+ * All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import type { Circular } from '~/routes/circulars/circulars.lib'
+
+export interface CircularAction {
+  initialize: () => Promise<object>
+  action: (circulars: Circular[], context: object) => Promise<void>
+  finalize: (context: object) => Promise<void>
+}
```

Comment:
Make this a generic type:

```suggestion
export interface CircularAction<T = unknown> {
  initialize: () => Promise<T>
  action: (circulars: Circular[], context: T) => Promise<void>
  finalize: (context: T) => Promise<void>
}
```

Reply:
Actually, that should be `export interface CircularAction<T = any>`.